src/pybind/mgr/orchestrator/module.py

```diff
@@ -604,10 +604,15 @@ def ukn(s: Optional[str]) -> str:
                 else:
                     refreshed = nice_delta(now, s.last_refresh, ' ago')
 
+                if s.spec.service_type == 'osd':
```

Comment:
@sebastian-philipp shouldn't it be :
```suggestion
                if s.spec.service_type == 'osd' or service_type == 'osd':
```
?

Reply:
otherwise, `ceph orch ls --service_type=osd` will still return "N/X", isn't it?

Reply:
Hm. If `service_type == 'osd`, than we're only going to get osd service descriptions as well, right?

Reply:
ok i've just tested, forget about that, it looks good as it is.